code/modules/projectiles/guns/magic/wand.dm

```diff
@@ -9,6 +9,7 @@
 	can_charge = FALSE
 	max_charges = 100 //100, 50, 50, 34 (max charge distribution by 25%ths)
 	var/variable_charges = TRUE
+	var/wrest_chance = 0.82644628 //For use in prob(), about a 1 in 121 chance
```

Comment:
for something so specific, it'd be better as a define

Reply:
I was thinking a define too initially, but I thought that someone might want the option to change this value for any child classes. (I.E. change it to 0 to disable wresting).
However, if you want me to to just make a define for the number then set the prototype value to that define, I could do that, but it seems a bit redundant as this is the only place this value shows up currently.

Reply:
if someone wants to disable wresting, they can then make it a variable, but i doubt anyone would ever do that. keep it as a define.

Reply:
Okay, I've changed it to a define now.